src/metal.cs

```diff
@@ -1344,21 +1595,78 @@ partial interface MTLTexture : MTLResource {
 		[Export ("iosurfacePlane")]
 		nuint IOSurfacePlane { get; }
 
-		[NoiOS, NoTV, Mac (10, 14)]
+		[iOS (13,0), TV (13,0), Mac (10, 14)]
 #if XAMCORE_4_0
 		[Abstract]
 #endif
 		[Export ("shareable")]
 		bool Shareable { [Bind ("isShareable")] get; }
 
-		[NoiOS, NoTV, Mac (10, 14)]
+		[iOS (13,0), TV (13,0), Mac (10, 14)]
 #if XAMCORE_4_0
 		[Abstract]
 #endif
 		[return: NullAllowed]
 		[return: Release]
 		[Export ("newSharedTextureHandle")]
 		MTLSharedTextureHandle CreateSharedTextureHandle ();
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Unavailable (PlatformName.MacCatalyst)]
+		[NoMac, NoTV, iOS (13, 0)]
+		[Advice ("This API is not available when using UIKit on macOS.")]
+		[Export ("firstMipmapInTail")]
+		nuint FirstMipmapInTail { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Unavailable (PlatformName.MacCatalyst)]
+		[NoMac, NoTV, iOS (13, 0)]
+		[Advice ("This API is not available when using UIKit on macOS.")]
+		[Export ("tailSizeInBytes")]
+		nuint TailSizeInBytes { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Unavailable (PlatformName.MacCatalyst)]
+		[NoMac, NoTV, iOS (13, 0)]
+		[Advice ("This API is not available when using UIKit on macOS.")]
+		[Export ("isSparse")]
+		bool IsSparse { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Mac (10, 15), TV (13,0), iOS (13, 0)]
+		[Export ("swizzle")]
+		MTLTextureSwizzleChannels Swizzle { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[Mac (10,15), TV (13,0), iOS (13,0)]
+		[Export ("newTextureViewWithPixelFormat:textureType:levels:slices:swizzle:")]
+		[return: NullAllowed]
+		IMTLTexture Create (MTLPixelFormat pixelFormat, MTLTextureType textureType, NSRange levelRange, NSRange sliceRange, MTLTextureSwizzleChannels swizzle);
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[NoiOS, NoTV, Mac (10, 15)]
+		[NullAllowed, Export ("remoteStorageTexture")]
+		IMTLTexture RemoteStorageTexture { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[NoiOS, NoTV, Mac (10,15)]
+		[Export ("newRemoteTextureViewForDevice:")]
+		[return: NullAllowed]
+		IMTLTexture Create (IMTLDevice device);
```

Comment:
same as before - the same argument(s) could be used for a non-remote texture leading to confusion